packages/showcase/src/app/utility-classes/spacing/metadata.ts

```diff
@@ -0,0 +1,78 @@
+import { ComponentMetadata } from "../../../common/types";
+import { ComponentTemplateItemText } from "../../../services/template-language/types";
+
+export const spacingMetadata = {
+  title: "Spacing",
+  description: "",
+  codeSnippets: {
+    blockInline: {
+      linkId: "spacing-body-inline-block",
+      title: "Block Inline",
+      template: {
+        tag: "div",
+        class: "spacing-body",
+        children: "<!-- Your content here -->"
+      }
+    },
+    block: {
+      linkId: "spacing-body-block",
+      title: "Block",
```

Comment:
Same for all titles.

```suggestion
      title: "2. Block",
```